app/controllers/trips_controller.rb

```diff
@@ -0,0 +1,61 @@
+class TripsController < ApplicationController
+  def index
+    if params[:index_trips_on_passenger_id]
+      @passenger =
+      Passenger.find(params[:passenger_id])
+      @trips = @passenger.trips
+    else
+      @trips = Trip.all
+    end
+  end
+
+  def new
+    @trip = Trip.new
+  end
+
+  def create
+    trip_attributes = {
```

Comment:
It might be worthwhile to set all of these defaults in a new model method rather than setting them all here